psiturk/psiturk_shell.py

```diff
@@ -1054,17 +1080,19 @@ def hit_create(self, numWorkers, reward, duration):
                              '    ________________________',
                              '    Total: $%.2f' % total])
             if self.sandbox:
-                print('  Ad URL: https://sandbox.ad.psiturk.org/view/%s?assignmentId=debug%s&hitId=debug%s&workerId=debug%s'
-                      % (str(ad_id), str(self.random_id_generator()), str(self.random_id_generator()), str(self.random_id_generator())))
-                print "Note: This url cannot be used to run your full psiTurk experiment.  It is only for testing your ad."
+                if use_psiturk_ad_server:
```

Comment:
It would be nice to still print the URL to the ad, just using the local server rather than the psiturk ad server.